### ISSUE TITLE
Move Apache exporter https://github.com/Lusitaniae/apache_exporter to community

### ISSUE THREAD

**Issue by frittentheke** [prometheus-community/community#47]
I'd like to propose moving the popular apache exporter to this community. I bluntly brought up the question with the creator @lusitaniae in issue https://github.com/Lusitaniae/apache_exporter/issues/145 and seems to like the idea.

With this being the de-facto standard exporter for Apache (see https://prometheus.io/docs/instrumenting/exporters/#http) already and Apache still being all over the place I believe it would be great addition.

**Comment by roidelapluie** [comment]
Do you have specific concerns with the apache exporter being independent? I wonder if we should move the exporter to the community without knowing what the future would bring - the exporter could be replaced by native solutions like https://github.com/ConSol/apache_mod_prometheus_status in the future.

I'd also note that I have commit access to the apache_exporter, even if I don't actively maintain it at the moment.

**Comment by frittentheke** [comment]
No there is no particular concern, but rather the impression that exporters in the community are aligned in their use of e.g. the exporter toolkit and features like support for TLS and auth. Nothing that cannot be achieved independently no, but I simply appreciate that the community exporters are treated in this way.

Thanks for the hint about https://github.com/ConSol/apache_mod_prometheus_status - having native metrics support without the need to run an exporter is nice - Just like HAProxy does it. But this still just being a module will likely make this an option, not a full replacement for an exporter to mod_status. Some installs simply might not be able or want to add modules to their Apaches.

**Comment by blkperl** [comment]
To echo what @frittentheke said I think there are benefits to having the exporter under the prometheus-community namespace and it allows the author to easily walk away from the project if they no longer wish to maintain it in the future without causing a potential disruption for users.

**Comment by frittentheke** [comment]
@Lusitaniae did say he was open to move the exporter over and also believes there could be benefits, see https://github.com/Lusitaniae/apache_exporter/issues/145#issuecomment-1347152613.

I understand @roidelapluie that prometheus-community cannot simply adopt ever exporter out there. But frankly the Apache webserver is really extremely common.

While I also believe in adding native Prometheus metrics support, as services like HAProxy, RabbitMQ and others did implment for their software and what would be possible with https://github.com/ConSol/apache_mod_prometheus_status this is still an "external" module (read: not something natively provided by Apache Httpd). So even some time down the road not everybody is comfortable adding such a module and rather use the well aged mod_status and scrape that scoreboard via an exporter.

**Comment by blkperl** [comment]
@roidelapluie @SuperQ any update on considering this request?

**Comment by blkperl** [comment]
Hey @SuperQ, I've managed the last four releases of https://github.com/Lusitaniae/apache_exporter/releases and my company no longer uses the exporter so it would be best if the community adopts it so that it can stay maintained.

**Comment by jkroepke** [comment]
@blkperl that are not good news. The community also needs responsible and active maintainers to keep the project up2date. Without that, they can't integrated into the prometheus community.

**Comment by SuperQ** [comment]
Yes, without someone to volunteer time to maintain things, we can not accept the project. I am not interested in sponsoring this myself.

**Comment by frittentheke** [comment]
When I opened this issue here @Lusitaniae was already willing to contribute the exporter.
@blkperl does it make sense to place a call for participation mentioning this issue here in the README in https://github.com/Lusitaniae/apache_exporter and maybe even do a release mentioning the current state of things in the release notes?

I mean there are 39 contributors and lots of users (I mean it's Apache httpd after all), there's got to be a small group of folks that would potentially help maintaining things?

**Comment by blkperl** [comment]
@frittentheke yes, can you open a PR to update the README with the call for participation?
